### DISCUSSION TITLE
Time format when showing all data in tooltip

### DISCUSSION THREAD

**Discussion by the21st** [vega/vega-lite#7616] [Q&A]
I want to set up a chart where the tooltip shows all fields of a datum. So I'm using

```json
  "mark": {
    "type": "bar",
    "tooltip": {"content": "data"}
  },
```

However, the time field in the tooltip shows what seems to be the timestamp in the epoch (seconds) format, which is just not very readable for humans:

![vegalite](https://user-images.githubusercontent.com/4610874/126964324-a34fb41a-a836-44d4-8996-1bf624d4e06d.png)

[This is the full spec](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAFgAY2AacaYsiygAlMiRoRQBmDgF9p3OgENCCvCCgKAtphQhFygLQAmAKwBOCTAUAjQ1bAAOY-YBsh+zBb2JCumzDGwB09rEDkQDQUAJwBrVUIATwAHbWQQKyiQbkIICAAbYkTVAkJMIh09FVluUsg6EiRUEAAPOKSUkBKNRIhIhVzMkHhMXLpypW0w+NbknQBHUgUiOGViNG1uIZGdTsKqkAASSjBGTAidMUJEymQAehu1hAUAOgRlxlIrJ7gIG6OTiPumEe+lyy0w9xYT3sTwAjE8AFaUaADCqUTCESiqCpGMwWay2BxOVzuTzeXz+QL2YJWFAAbVAenahjYMLY+hZ7JhABUOMheRwBjtJKYniw5AzxjpmazOZyeTC+WxFYLToVkBIRYZxbpJalpWyOSyeYZFcqsqrhdrGVKWQaYXK2BJTQLzV1JNDTFbdSB9bKjWwWM62Cq3erofYvSUbTLDdy2MYgyG1RJoQB2SNM21+uPOROu5NpjPRu0O1N5joWsNPZxFvVZ2M8+zloVV9OcCVRusx+3+0zNysap5a9s6zs++s97ms-uh9iw2vj7tymEK-nB-OsOESBe+hswk1rpOsQxPYw7ifLp2HjfIFgSIfnpdGmGB68V2f3s8j61dkvPhNvi2d5PNu37eruk5cjCuaAZWLAnm2HaZk+U5lrBs5woho7IX+U5NuhaosHCw5IcW2ZQX2BGsGwTyemBY4QXKzIzoRNFYT+i64VyhirkqLrvqxoqPlxhgHnx64CdRD70Th5GGFe4lHuqIp0aRv5+oYPLiWaknKdWwkafKLHCkJMlkYamlsGJOktoOoFqZxhmOsZekwgZFk8q+ik3hq7k9pZAHebpKa0X57KWTBQW2ap2Hmf5PJoVFA7hmFVmNi5IUxRxjFpWwlFJaGg5fg5OWadOVG3jRYpmepHkri5RGnqlZXWfxQEIc1UEKTZcH3tVJUXmlL4NfeEY1Y5dWBT1H76eNpVQZF02ESeWXgYNZWJUtm6hXN61QfhBWEXC-WxbV8UwvlW23nCbm7U+mnMRVc6rQxe08Q1NE1ndf4Pa1ElAVVnXyR90gALqyEAA) (it was generated by altair)

Is there anything I can do to format the time to a human-readable format, while keeping the `"tooltip": {"content": "data"}` option?

**Comment by kanitw** [comment]
I think this requires modification to the library, so I'm filing this issue https://github.com/vega/vega-lite/issues/7652.
